schema/synthetix/view_synths_addresses.sql

```diff
@@ -0,0 +1,10 @@
+CREATE OR REPLACE VIEW synthetix.view_synths_addresses (address, currencyKey) AS SELECT
+  address, currencyKey
+FROM synthetix."Issuer_evt_SynthAdded"
```

Comment:
will change this in the morning tomorrow to align with the fields on the event that Mats added :)